Documentation/official-docs.md

```diff
@@ -0,0 +1,6 @@
+# Official Microsoft Docs
+
+Microsoft maintains official documentation for .NET. You can see a list of those below:
+
+- [.NET Framework 4.6+](https://msdn.microsoft.com/library/w0x726c2(v=vs.110).aspx)
+- [.NET Core](https://docs.microsoft.com/dotnet/)
```

Comment:
https://docs.microsoft.com/dotnet/ links to documentation for both .Net Core and .Net Framework.

So, saying that it's just for .Net Core is inaccurate and I think that linking to .Net Framework documentation too is redundant.